### PR TITLE
UHM-6254: View/Edit page for saved forms -- Breadcrumbs

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/visit/visit.js [hunk 1]
```diff
@@ -297,6 +297,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                             visit: $scope.visit,
                             encounter: $scope.encounter,
                             patient: $scope.visit.patient,
+                            breadcrumbOverride: encodeURIComponent(JSON.stringify(breadcrumbOverride)),
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/visit.js:300]: Not directly related to this commit, but fixes an issue where breadcrumbs weren't working right in the vust note "section" context

File: omod/src/main/webapp/resources/scripts/visit/visit.js [hunk 2]
```diff
@@ -593,6 +594,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                                 patient: {
                                     uuid: patientUuid
                                 },
+                                breadcrumbOverride: encodeURIComponent(JSON.stringify(breadcrumbOverride)),
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/visit.js:597]: Not directly related to this commit, but fixes an issue where breadcrumbs weren't working right in the vust note "section" context

File: omod/src/main/webapp/resources/scripts/visit/visit.js [hunk 3]
```diff
@@ -730,6 +732,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                         returnUrl: encodeURIComponent(
                           window.location.pathname
                           + "?visit=" + $scope.visit.uuid
+                          + "&encounter=" + encounter.uuid
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/visit.js:735]: Fixed a bug where when you clicked "Cancel" while edited a form you lost the "Saved Form" context

File: omod/src/main/webapp/resources/scripts/visit/visit.js [hunk 4]
```diff
@@ -747,6 +750,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                         returnUrl: encodeURIComponent(
                             window.location.pathname
                             + "?visit=" + $scope.visit.uuid
+                          + "&encounter=" + encounter.uuid
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/visit.js:753]: Fixed a bug where when you clicked "Cancel" while edited a form you lost the "Saved Form" context

**Review by mseaton — APPROVED** [review]
Nice, lgtm, and bonus to get those bugs fixed

**Pull request merged**